build.gradle

```diff
@@ -125,6 +125,34 @@ publishing {
                 }
 
                 developers {
+                    developer {
+                        name = "Civitaspo"
+                        email = "civitaspo@gmail.com"
+                    }
+                    developer {
+                        name = "Satoshi Akama"
+                        email = "satoshiakama@gmail.com"
+                    }
+                    developer {
+                        name = "Naotoshi Seo"
+                        email = "sonots@gmail.com"
+                    }
+                    developer {
+                        name = "Khoa Nguyen"
+                        email = "instcode@gmail.com"
+                    }
+                    developer {
+                        name = "Robert Nguyen"
+                        email = "ng.hung83@gmail.com"
+                    }
```

Comment:
@tvhung83 We're nominating you in `pom.xml`'s `<developers>` like we did in https://github.com/embulk/embulk-input-jdbc/pull/210#discussion_r639414980